tools/workspace-plugin/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+const path = require('node:path');
+
+module.exports = {
+  extends: ['plugin:@fluentui/eslint-plugin/node'],
+  root: true,
+  ignorePatterns: ['!**/*'],
+  rules: {
+    'import/no-extraneous-dependencies': 'off',
+    '@typescript-eslint/no-unused-vars': ['error', { varsIgnorePattern: '^_', argsIgnorePattern: '^_' }],
+  },
+  overrides: [
+    {
+      files: '*.json',
+      parser: 'jsonc-eslint-parser',
+      rules: {
+        '@typescript-eslint/naming-convention': 'off',
+      },
+    },
+    {
+      files: ['*.ts', '*.tsx', '*.js', '*.jsx'],
+      rules: {},
+    },
+    {
+      files: ['*.ts', '*.tsx'],
+      parserOptions: {
+        project: [path.join(__dirname, 'tsconfig.lib.json'), path.join(__dirname, 'tsconfig.spec.json')],
```

Comment:
because of this: `Error: You have attempted to use the lint rule "@typescript-eslint/await-thenable" which requires the full TypeScript type-checker to be available, but you do not have "parserOptions.project" configured to point at your project tsconfig.json files in the relevant TypeScript file "overrides" block of your ESLint config "tools/workspace-plugin/.eslintrc.js`

Reply:
this is known bug which needs a complete rework , I'm ok having this failing on local machine as you can avoid this via `--no-verify`, see more details https://github.com/microsoft/fluentui/issues/31751

lets revert this one

Reply:
aight, thanks for the info 👍🏻